secboot/secboot_tpm_test.go

```diff
@@ -821,6 +823,11 @@ func (s *secbootSuite) TestResealKey(c *C) {
 		{tpmEnabled: true, resealErr: mockErr, resealCalls: 1, expectedErr: "some error"},
 		{tpmEnabled: true, resealCalls: 1, expectedErr: ""},
 	} {
+		mockAuthKey := []byte{1, 3, 3, 7}
+		mockAuthKeyFile := filepath.Join(c.MkDir(), "auth-key-file")
```

Comment:
AIUI this file is called "tpm-policy-auth-key-file" in the real code. Should we try to be consistent with the naming? Also for the vars, i.e. mockTpmPolicyAuthKey instead of mockAuthKey? If it's the same file/mechanism it would certainly help me as someone who is not that deep into this code to find familiar file/var names :) But no superstrong opinion here if you/Samuele feel differently it's fine.